src/components/common/InAppBrowserWarningModal.tsx

```diff
@@ -49,13 +50,14 @@ const InAppBrowserWarningModal = () => {
   if (isLineInAppBrowser) window.location.assign(lineOpenExternalBrowserUrl)
 
   const isRemind = sessionStorage.getItem('kolable.InAppBrowserWarning.isRemind')
-  if (isInAppBrowser && !isRemind) {
+  if (isInAppBrowser && !isToast && !isRemind) {
     toast({
       title: formatMessage(commonMessages.InAppBrowserWarningModal.warning),
       status: 'warning',
       isClosable: true,
       position: 'bottom',
     })
+    setIsToast(true)
```

Comment:
This checker is only used once
could use a `let` variable instead.

Reply:
When using 'let,' it gets cleared upon re-rendering.

Reply:
[`preventing-duplicate-toast`](https://chakra-ui.com/docs/components/toast#preventing-duplicate-toast)

Reply:
Thanks a lot for your solution!